### DISCUSSION TITLE
infrastructure/proxmox/gpu-passthrough/igpu-split-passthrough/

### DISCUSSION THREAD

**Discussion by giscus** [fire1ce/3os.org#111] [General]
# infrastructure/proxmox/gpu-passthrough/igpu-split-passthrough/

Proxmox iGPU split passthrough to VM configuration for hardware acceleration.

https://3os.org/infrastructure/proxmox/gpu-passthrough/igpu-split-passthrough/

**Comment by M1NiSTrY-GiT** [comment]
Thank you for this awesome & well written guide. My Plex server now happily uses the iGPU for transcoding :-)

**Comment by UntouchedWagons** [comment]
You can use the q35 machine type for the VM too, the use of i440fx is not required.

**Reply by UntouchedWagons** [reply]
Oh also in modern installs of proxmox you need to edit `/etc/kernel/cmdline` instead of `/etc/default/grub` for the kernel parameters.

**Reply by ruanmed** [reply]
@UntouchedWagons, You sure? I don't think you should be editing `/etc/kernel/cmdline` directly in all systems. 

I think the suggest way is to update `/etc/default/grub` and then run `update-grub` command that will automatically verify if you are using EFI and update `/etc/kernel/cmdline` accordingly.

**Reply by UntouchedWagons** [reply]
A minor error on my part, when installing Proxmox and choosing ZFS, GRUB is not used as a bootloader, systemd-boot is. So editing `/etc/default/grub` will have no effect.

**Comment by dderbidge** [comment]
Does this guide work for Proxmox 8.2? Part of the instructions say to set the Mdev type for the PCI device, but the look and feel of this new Proxmox version is different and it doesn't let you set the Mdev type. The drop-down box is grayed out as if there's another step needed before you can even use it.

**Comment by auggy** [comment]
all the screen shots in this guide differ from the proxmox i just downloaded. I would consider taking this page down or at least specify what version to use.

**Reply by auggy** [reply]
edit: i noticed this was edited a week ago, did proxmox change that much in that time?

**Comment by chiendo97** [comment]
For some reasons, I have to manually load driver by running `sudo modprobe i915` to see renderD128 in /dev/dir in Ubuntu 24.

**Reply by stevefxp1** [reply]
Did you blacklist the driver within the Grub parameters? If yes I believe you have to remove it.

**Comment by stevefxp1** [comment]
I would assume that the same Grup parameters are used for SystemD implementations?
